homework.js

```diff
@@ -1,26 +1,86 @@
+
+
+
 // Problem 1
 // declare a variable called fruit
 // assign your variable the value 6
 // console log your variable
 
+let fruit = 6;
+console.log(fruit);
+
+
+
+// declare a variable called fruit
+// assign your variable the value "pineapple"
+// console log your variable
+
+//let fruit = "pineapple";
+//console.log(fruit);
+
 
 // write a function called product that accepts two numbers and 
 // logs the product in the console and then
 // returns the answer when you multiply them both together
 // call that function on the numbers 4 & 689 to test it
 
+function product(number1, number2) {
+  let solution = number1 * number2;
+  console.log(solution);
+  return solution;
+}
+product(4, 689);
+
+
 // Problem 2
 // write a function called evens that takes an array of numbers, removes 
 // all the numbers that are odd and returns a new array with only the even numbers
 // remember that even numbers have a remainder of 0 when you divide them by 2
 // call your function on this array of numbers [4, 37, 18, 29, 52, 103, 22]
 
+function evens(array) {
+  //let evenArray = [];
+  let remainder = number % 2;
```

Comment:
You perform this operation later on every number in the array, which is what you want.